cico_functions.sh

```diff
@@ -44,9 +65,18 @@ function install_deps() {
     git
 
   service docker start
+  
+  #Enable qemu and binfmt support
+  docker run --rm --privileged docker/binfmt:66f9012c56a8316f9244ffd7622d7c21c1f6f28d
+  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+  
   echo 'CICO: Dependencies installed'
 }
 
+function version() {
+  printf '%02d' $(echo "$1" | tr . ' ' | sed -e 's/ 0*/ /g') 2>/dev/null
```

Comment:
Does this code always work like expected?

For me:
$ uname -a
5.5.17-200.fc31.x86_64

result of  function:

$  printf '%02d' $(echo "$(uname -r)" | tr . ' ' | sed -e 's/ 0*/ /g') 2>/dev/null
0505170000

Maybe we could simple split by '-':

$ kernel_version=$(uname -r); echo ${kernel_version%%-*}
5.5.17

 What do you think?

Reply:
We wanted comparison between 2 whole numbers, checking whichever is greater, for this reason we added this check. This code is working as per our expectation. Do you sense any possible issue in this?

**Code :-**
```
# Kernel
kernel_version="$(uname -r)"
if [[ "${kernel_version%%-*}" < "4.8" ]]; then
        echo >&2 "Kernel $kernel_version too old - need >= 4.8." \
                " Install a newer kernel."
else
        echo >&2 "kernel $kernel_version has binfmt_misc fix-binary (F) support."
fi
```

If kernel Version = 3.10.0-1062.el7 (**Works as expected**)
Condition : 3.10.0 < 4.8
O/P : Kernel 3.10.0-1062.el7 too old - need >= 4.8.  Install a newer kernel.

If Kernel Version = 4.15.0-66-generic (**Doesn't work as expected**)
Condition : 4.15.0 < 4.8
O/P : Kernel 4.15.0-66-generic too old - need >= 4.8.  Install a newer kernel.

If kernel Version = 5.5.17-200.fc31 (**Works as expected**)
Condition : 5.5.17 < 4.8 
O/P : kernel 5.5.17-200.fc31.x86_64 has binfmt_misc fix-binary (F) support.

Reply:
Thanks, it's make sense.

Reply:
@AndrienkoAleksandr Could you please tell me whether you are testing these changes on env similar to ci.centos?

Reply:
@amisevsk Thanks for the suggestion. Implemented version check using `sort` with `-V`. Please review.